### ISSUE TITLE
Would COLMAP discard too similar images?

### ISSUE THREAD

**Issue by jytime** [colmap/colmap#2584]
Hi,

I remembered read it somewhere that colmap will discard/drop too similar images during reconstruction, but failed to find this in the recent source code. May I ask is this still true in the current colmap version? 

Best,
Jianyuan

**Comment by ahojnnes** [comment]
No, colmap does not automatically drop redundant images and this has never been the case.
